modules/storage-persistent-storage-efs-configmap.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-creating-configmap_{context}"]
+= Creating a ConfigMap
+
+It is recommended to use a ConfigMap to contain all of the environment
+variables required for the EFS provisioner.
+
+.Procedure
+
+. Define an {product-title} ConfigMap containing the environment
+variables by creating a `configmap.yaml` with the following contents:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: efs-provisioner
+data:
+  file.system.id: <file-system-id> <1>
+  aws.region: <aws-region> <2>
+  provisioner.name: openshift.org/aws-efs <3>
+  dns.name: "" <4>
+----
+<1> Defines the Amazon EFS file system ID.
```

Comment:
Check the references to Amazon in this section. They might need to be AWS instead of Amazon. For example, I know that it should be AWS region instead of Amazon region.